public/wp-content/themes/ColoredCow/mailer.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+require_once __DIR__ . '/../../plugins/wpmandrill/lib/mandrill.class.php';
```

Comment:
use plugins directory path. There should be an inbuilt WP function to get the plugins directory.